scripts/deploy.sh

```diff
@@ -43,6 +43,10 @@ node -e "var package = require('./package.json'); \
   package['jsnext:main'] = 'index.js'; \
   package['react-native'] = 'index.js'; \
   package.typings = 'index.d.ts'; \
+  package.exports['.'].module = './index.js'; \
+  package.exports['.']['module-sync'] = './index.js'; \
+  package.exports['.'].require = './bundle.umd.js'; \
```

Comment:
The problem here is the `.umd.js` extension.

<img width="1686" height="559" alt="image" src="https://github.com/user-attachments/assets/800322dc-ed3c-4077-a8a5-6ea0cd28f03c" />

As weird as it sounds, the name of this file would need to be 
```diff
-./bundle.umd.js
+./bundle.umd.cjs
```
for this to work.

Reply:
Thanks for testing @phryneas! My first reaction is to go with the `.cjs` extension, because I *believe* that tagging this with `”type”: “module”` is considered best practice & it sounds like an easy fix, but I don’t actually know for sure.

What do you think @jerelmiller ?